### PR TITLE
cli: implement wallet import/export functionality

### PR DESCRIPTION
**Pull request by fyrchik**
Support some of the capabilities outlined in #26 .
- import WIF
- import WIF + multisig
- export WIF

Fix some minor bugs.

### CODE REVIEW

File: cli/wallet/wallet.go [hunk 1]
```diff
@@ -72,10 +73,71 @@ func NewCommands() []cli.Command {
 					},
 				},
 			},
+			{
+				Name:   "import-multisig",
+				Usage:  "import multisig contract",
```
Comment (roman-khimov) [inline cli/wallet/wallet.go]: I think we need more extensive help for this one, it's not obvious how to pass multisig keys there.

File: cli/wallet/wallet.go [hunk 2]
```diff
@@ -54,10 +59,16 @@ func NewCommands() []cli.Command {
...
-					cli.BoolFlag{
-						Name:  "decrypt, d",
-						Usage: "Decrypt encrypted keys.",
-					},
+					decryptFlag,
+				},
+			},
+			{
+				Name:   "export",
+				Usage:  "export keys for address",
```
Comment (roman-khimov) [inline cli/wallet/wallet.go]: Again, positional arguments need additional comments right in the help section. Users are gonna be invoking `help` to find out how to use this thing and it should tell them how.

**Comment by roman-khimov** [comment]
One thing we're missing here: `newAccountFromPrivateKey` should also generate a simple `CHECKSIG` contract, that actually was one of the issues with our testnet CN, it had a key in the wallet, but it didn't have a contract accociated with it and then `GetAccount()` failed to find the key because its operation is based on contracts.

**Comment by roman-khimov** [comment]
Other than that it's fine, we may probably tune the `cli` part of it, but the main thing is that all of the functionality is there.

**Comment by fyrchik** [comment]
@roman-khimov isn't it somehow different from what is done in 5a727cabf8e ?

File: cli/wallet/wallet.go (hunk 2 above)
Comment (fyrchik) [reply cli/wallet/wallet.go]: Added `UsageText` which is shown when help for a specific command is requested.

File: cli/wallet/wallet.go (hunk 1 above)
Comment (fyrchik) [reply cli/wallet/wallet.go]: Added `UsageText`.

**Comment by roman-khimov** [comment]
Yeah, that's what it is, somehow I've missed that this change actually fixes `newAccountFromPrivateKey`.

**Pull request merged**